### PR TITLE
no more all on one page

### PR DESCRIPTION
**Pull request by asiagromasia**
changed the route for loging patient and  provider

Co-Authored-By: Oyunomin Munkhkhurel <oyunominjuno@users.noreply.github.com>

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -18,39 +18,45 @@ import PatientImmediateAttList from './components/PatientImmediateAttList.compon
...
+      </div>
       <div className="container">
-          <Route path="/patient/login">
+          <Route exact path="/patient/login">
               <Login />
           </Route>
        </div>
-          <Route path="/provider/login">
+      <div className="container">
+          <Route exact path="/provider/login/:id">
```
Comment (All-Schuck-Up) [inline client/src/App.js:34]: I do not think /:id needs to be here

File: client/src/App.js [hunk 2]
```diff
@@ -18,39 +18,45 @@ import PatientImmediateAttList from './components/PatientImmediateAttList.compon
...
               <LoginProvider />
           </Route>
-         <Route path="/patient/:id">
+      </div>
+      <div className="container">
+         <Route exact path="/patient/login/:id">
```
Comment (All-Schuck-Up) [inline client/src/App.js:39]: This route also could be altered to not reflect login

File: client/src/App.js (hunk 1 above)
Comment (asiagromasia) [reply client/src/App.js:34]: we need this parameter to display details

Comment (All-Schuck-Up) [reply client/src/App.js:34]: I believe that is the function of the backend routes

**Comment by yergu** [comment]
Every thing looks good

**Pull request merged**